Visual Studio/Realm.vcxproj

```diff
@@ -363,7 +363,8 @@
       <PrecompiledHeader>
       </PrecompiledHeader>
       <DisableSpecificWarnings>4355;4996</DisableSpecificWarnings>

```

Comment:
I agree that we should try to not change 3rd party code. Can we then re-enable these warnings?